### PR TITLE
feat: 📦 : Guide Component Added for Issue #85

### LINKED ISSUE #85: Guid Section
Related to #78 

Image

### PR DESCRIPTION
**Pull request by urmirahman**
# Description
Guide Component added  for #78

Fixes #85 

## Type of change

Please delete options that are not relevant.

- [x] New feature (non-breaking change which adds functionality)
- [x] Hero Section components are re-used with very few change
- [x] Storybook is also implemented
- [x] All the data's are coming from akij-cms.

### No test has been done

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my code
- [x] I have commented my code, particularly in hard-to-understand areas

# Output in Localhost:3010
Guide Section is similar to our Product section. Here is the output.

guidesection

Closes #85

### CODE REVIEW

File: apps/akij-food-beverage/src/components/ui/features/Guide/Products/Products.tsx [hunk 1]
```diff
@@ -0,0 +1,28 @@
+import { Alert, CircularProgress, Grid, Typography } from '@mui/material'
+import { AnyMxRecord } from 'dns'
+import type { NextPage } from 'next'
+
+import type { CmsProducts } from '~/types/types'
+import ProductItem from './ProductItem'
+
+const Products = ({ products, error, loading }: any) => {
```
Comment (TanvirAlam) [inline apps/akij-food-beverage/src/components/ui/features/Guide/Products/Products.tsx:8]: why dont you re-use the one we already have? you are creating the same component as this one: `src/components/features/Hero/Products.tsx`?

**Review by TanvirAlam — CHANGES_REQUESTED** [review]
LGTM :+1: I like, think re-usability

File: apps/akij-food-beverage/src/components/ui/features/Guide/Products/Products.tsx (hunk 1 above)
Comment (urmirahman) [reply apps/akij-food-beverage/src/components/ui/features/Guide/Products/Products.tsx:8]: need to add some style classes amd removed extra props divs.. this component is not entirely same as the hero/product Should have changed the component name??

**Comment by urmirahman** [comment]
@TanvirAlam  I think all the conflicts have been removed . Please check.

**Review by TanvirAlam — APPROVED** [review]
LGTM :+1:

**Pull request merged**